src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -939,12 +939,21 @@ public Mono<Collection<String>> getExportFormats() {
             .bodyToMono(typeRef);
     }
 
-    public Mono<ExportNetworkInfos> exportNetwork(UUID studyUuid, String format) {
-        Mono<UUID> networkUuidMono = networkStoreService.getNetworkUuid(studyUuid);
+    public Mono<ExportNetworkInfos> exportNetwork(UUID studyUuid, UUID nodeUuid, String format) {
 
-        return networkUuidMono.flatMap(uuid -> {
-            String path = UriComponentsBuilder.fromPath(DELIMITER + NETWORK_CONVERSION_API_VERSION + "/networks/{networkUuid}/export/{format}")
-                .buildAndExpand(uuid, format)
+        if (!networkModificationTreeService.getStudyRootNodeUuid(studyUuid).equals(nodeUuid)
```

Comment:
Use reactive assert like other requets ?

Reply:
Done, but could you check if it's ok like that ?